Base/Log.cs

```diff
@@ -164,6 +164,20 @@ public LogAppenderFile()
                 File.Delete(FileName);
         }
 
+        public void CopyToClipboard()
```

Comment:
I put this here instead of in MainForm.cs so the knowledge of where the file is on disk is contained within the log class.